tests/common.py

```diff
@@ -783,21 +783,38 @@ class MockModule:
 
     def __init__(
         self,
-        domain=None,
-        dependencies=None,
-        setup=None,
-        requirements=None,
-        config_schema=None,
-        platform_schema=None,
-        platform_schema_base=None,
-        async_setup=None,
-        async_setup_entry=None,
-        async_unload_entry=None,
-        async_migrate_entry=None,
-        async_remove_entry=None,
-        partial_manifest=None,
-        async_remove_config_entry_device=None,
-    ):
+        domain: str | None = None,
+        *,
+        dependencies: list[str] | None = None,
```

Comment:
Small tweak here => allow only domain to be a positional argument.
I think it makes it clearer what the purpose of the "second" argument is.
This only affects `dependencies`, as all other arguments are already passed via keyword.